roles/debian-hardening/vars/main.yml

```diff
@@ -20,3 +20,6 @@ hardened_services:
   - pacemaker
   - syslog-ng
   - ceph-crash
+
+grub_password: grub.pbkdf2.sha512.65536.401EFA213B7334C4E6975DE70AEF0A61B35F64AD5516C0B111083DC7D067C73634C017E44B4B9BE65AAE74F61ACD31A3D52BCD3FE5FBF7BDFE6431C60D799BF762EBB5AC98C47839386E0503F626BD7BDA185EE3C64B1A77B630577C6CB333C5D222E943F8EFB6E4846CDADC1AEDF6B9DEE1EAEEAA606333B87AA14D6CF60D4A6D9CC3CF69C1F88EC86596A2B50174EB1A844A296B6950D6409A758FBF097B72E41736ADF1D8F3C3CAC42A3CB479A5DDABE2B08B7C4634691FCBCAB569596FE1B97C6F6085EC7D698C31C2C4350779B78DE445D12B02279FE6DD92AF1A66C48A7E88030865DEE9AC7F2D339774A1494ED3E025B9C8ECAFE9D9FECA54CA46F3F3.CCB479FE2FF11614F2CBDD687B467BB776B0B317543797C59AC06897975441433F83E332A8F72B9DE96FEDB27E5E01C98663576637E6E045759DB438DD47B2FC 
```

Comment:
As discussed, this shouldn't be hardcoded. Add and document an ansible variable to the inventories.